packages/shorebird_cli/test/src/cache_test.dart

```diff
@@ -86,33 +97,42 @@ void main() {
 
     group('clear', () {
       test('deletes the cache directory', () async {
-        Cache.shorebirdCacheDirectory.createSync(recursive: true);
-        expect(Cache.shorebirdCacheDirectory.existsSync(), isTrue);
-        cache.clear();
-        expect(Cache.shorebirdCacheDirectory.existsSync(), isFalse);
+        runWithOverrides(() {
+          Cache.shorebirdCacheDirectory.createSync(recursive: true);
+          expect(Cache.shorebirdCacheDirectory.existsSync(), isTrue);
+          cache.clear();
+          expect(Cache.shorebirdCacheDirectory.existsSync(), isFalse);
+        });
       });
 
       test('does nothing if directory does not exist', () {
-        expect(Cache.shorebirdCacheDirectory.existsSync(), isFalse);
-        cache.clear();
-        expect(Cache.shorebirdCacheDirectory.existsSync(), isFalse);
+        runWithOverrides(() {
+          expect(Cache.shorebirdCacheDirectory.existsSync(), isFalse);
+          cache.clear();
+          expect(Cache.shorebirdCacheDirectory.existsSync(), isFalse);
+        });
       });
     });
 
     group('updateAll', () {
       group('patch', () {
         test('downloads correct artifacts', () async {
-          expect(cache.getArtifactDirectory('patch').existsSync(), isFalse);
-          await expectLater(cache.updateAll(), completes);
-          expect(cache.getArtifactDirectory('patch').existsSync(), isTrue);
+          await runWithOverrides(() async {
+            expect(cache.getArtifactDirectory('patch').existsSync(), isFalse);
+            await expectLater(cache.updateAll(), completes);
+            expect(cache.getArtifactDirectory('patch').existsSync(), isTrue);
+          });
         });
 
         test('pull correct artifact for MacOS', () async {
           when(() => platform.isMacOS).thenReturn(true);
           when(() => platform.isWindows).thenReturn(false);
           when(() => platform.isLinux).thenReturn(false);
 
-          await expectLater(cache.updateAll(), completes);
+          await runWithOverrides(
```

Comment:
minor but it would be nice to standardize on whether we wrap just the call with `runWithOverrides` or the entire expectation. I prefer just the call since that's what we've done elsewhere wdyt?

Reply:
Fixed.